js/chown.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
+import * as flatbuffers from "./flatbuffers";
+import * as msg from "gen/cli/msg_generated";
+import * as dispatch from "./dispatch";
+
+function req(
+  path: string,
+  uid: number,
+  gid: number
+): [flatbuffers.Builder, msg.Any, flatbuffers.Offset] {
+  const builder = flatbuffers.createBuilder();
+  const path_ = builder.createString(path);
+  const inner = msg.Chown.createChown(builder, path_, uid, gid);
+  return [builder, msg.Any.Chown, inner];
+}
+
+/**
+ * Change owner of a regular file or directory synchronously, Unix only at the moment
```

Comment:
Adjust the punctuation to be this:

Change owner of a regular file or directory synchronously. Unix only at the moment.